packages/plugin-protocol-http/src/http.ts

```diff
@@ -187,6 +199,7 @@ export default ({ agent }: { agent?: { http: Agent; https: Agent } } = {}): Plug
                     ...query,
                 }),
                 {
+                    ...requestOptions,
```

Comment:
We have plans to move on undici, so this will create problem for us, because users can use internal node-fetch API and we will have problem to migrate to udici

Reply:
I've removed this commit as we decided to not implement such functionality